pkg/operator/controller/garden/garden/reconciler_reconcile.go

```diff
@@ -606,15 +677,20 @@ func (r *Reconciler) deployEtcdsFunc(garden *operatorv1alpha1.Garden, etcdMain,
 				backupLeaderElection = r.Config.Controllers.Garden.ETCDConfig.BackupLeaderElection
 			}
 
-			container, prefix := etcdConfig.Main.Backup.BucketName, "virtual-garden-etcd-main"
-			if idx := strings.Index(etcdConfig.Main.Backup.BucketName, "/"); idx != -1 {
-				container = etcdConfig.Main.Backup.BucketName[:idx]
-				prefix = fmt.Sprintf("%s/%s", strings.TrimSuffix(etcdConfig.Main.Backup.BucketName[idx+1:], "/"), prefix)
+			container, prefix := etcdMainBackupBucketName(garden), "virtual-garden-etcd-main"
+			if idx := strings.Index(container, "/"); idx != -1 {
```

Comment:
This use case is not supported anymore. The `BackupBucket` creation would fail because of an invalid object name.

Reply:
Backup bucket is now created also in the case that the bucket name is specified. E2E test has been adapted to not use slashes anymore.